scripts/alert_emails/campaign-monitor.ts

```diff
@@ -40,6 +30,30 @@ export interface CampaignMonitorMessageSent {
   MessageID: string;
 }
 
+class CampaignMonitorException implements CampaignMonitorError {
+  Code: number;
+  Message: string;
+  constructor(code: number, message: string) {
+    this.Code = code;
+    this.Message = message;
+  }
+}
+
+function isStatusOK(status: number) {
+  return 200 <= status && status < 300;
+}
+
+function isRateLimitExceeded(status: number) {
+  return status === 429;
+}
+
+export function isInvalidEmailError(error: CampaignMonitorError) {
+  return (
+    error.Code === 1 &&
+    error.Message === 'A valid recipient address is required'
```

Comment:
nit: This is existing code, but we should probably only check `Code` not `Message` since the latter probably isn't guaranteed to never change.

CM [documents](https://www.campaignmonitor.com/api/transactional/#send-classic-email) code 1 as "Invalid Email Address" so we don't need to check the message.